src/Client.js

```diff
@@ -1037,13 +1037,13 @@ class Client extends EventEmitter {
      * @returns {Promise<Object|null>}
      */
     async getNumberId(number) {
-        if (!number.endsWith('@c.us')) {
-            number += '@c.us';
-        }
+        //if (!number.endsWith('@c.us')) {
+           // number += '@c.us';
+        //}
 
         return await this.pupPage.evaluate(async number => {
-            const wid = window.Store.WidFactory.createWid(number);
-            const result = await window.Store.QueryExist(wid);
+            //const wid = window.Store.WidFactory.createWid(number);
+            const result = await window.Store.QueryExist('phone',`+${number}`);
```

Comment:
Maybe check if it already starts with "+" if not appent "+"

Reply:
Ok